.coveragerc

```diff
@@ -0,0 +1,25 @@
+[run]
+# Include specific source so we can tell when entire directories are not measured
+# XXX It'd sure be nice to have a way to make this repo-agnostic.
+source = 
+    coded_tools
+    decomposer
+    maker_playground
```

Comment:
Experiment with adding a .coveragerc file to show test coverage (or lack thereof) upon pytest output.
So far I have not found a repo-agnostic way to include source in all top-level directories.
Giving up on that for now, but will adopt this because it makes the tests.yml for GitHub actions completely generic.